exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -37,10 +38,16 @@ var testData = []struct {
 		""},
 	{4, 10, Product{}, Product{}, "No palindromes"},
 	{10, 4, Product{}, Product{}, "fmin > fmax"},
-	/* bonus curiosities.  (can a negative number be a palindrome?
-	// most say no
+}
+
+// Bonus curiosities. Can a negative number be a palindrome? Most say no
```

Comment:
Can we have one comment per test case in the bonus section so that each comment is close to the test case that it refers to?